oslib/libmtd_common.h

```diff
@@ -69,7 +69,7 @@ extern "C" {
 #endif
 
 /* define a print format specifier for off_t */
-#ifdef __USE_FILE_OFFSET64
+#ifdef _LARGEFILE_SOURCE
```

Comment:
Is this correct? i.e. on a 32 bit systems will you definitely make all the file functions take 64 bits offsets?

Reply:
It is correct. The Makefile currently passes -D_LARGEFILE_SOURCE. This change merely fixes format strings.

Reply:
Excellent point. Since the Makefile does so unconditionally does it make more sense to do away with the #ifdef branches?

Reply:
In fact, where do `PRIxoff_t` and `PRIdoff_t` get used in fio?